torch/csrc/xpu/Module.cpp

```diff
@@ -11,24 +11,30 @@
 #include <torch/csrc/utils/python_numbers.h>
 #include <torch/csrc/utils/python_strings.h>
 
+#ifndef WIN32
```

Comment:
Wasn't this copied from the cuda version? Why is this needed here but not there?

Reply:
`pthread.h` is not supported on Windows. It's the same with other backends:
https://github.com/pytorch/pytorch/blob/b704c7cf0f8fb8b92198ffd4cd07716a7b9efae1/torch/csrc/cuda/Module.cpp#L53-L55

https://github.com/pytorch/pytorch/blob/b704c7cf0f8fb8b92198ffd4cd07716a7b9efae1/torch/csrc/mtia/Module.cpp#L11-L13